solidus.gemspec

```diff
@@ -14,6 +14,8 @@ Gem::Specification.new do |s|
   s.homepage    = 'http://solidus.io'
   s.license     = 'BSD-3-Clause'
 
+  s.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
I think this is needed in all the other subgems as well, isn't it?

Reply:
Oh, okay, I'll update the subgems too :+1: TODO